community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -199,8 +199,9 @@ object projects:
 
   lazy val stdLib213 = SbtCommunityProject(
     project       = "stdLib213",
+    extraSbtArgs  = List("-Dscala.build.compileWithDotty=true"),
     sbtTestCommand   = """;set scalacOptions in Global += "-Yerased-terms" ;library/compile""",
-    extraSbtArgs  = List("-Dscala.build.compileWithDotty=true")
+    sbtPublishCommand = """;set scalacOptions in Global += "-Yerased-terms" ;set publishArtifact in (library, Compile, packageDoc) := false ;library/publishLocal""",
```

Comment:
`-Yerased-terms` should already be set when `-Dscala.build.compileWithDotty` is enabled, is that not working for some reason? https://github.com/scala/scala/blob/8c86b7d7136839538cca0ff8fca50f59437564c0/project/DottySupport.scala#L51

Reply:
I didn't add it for any better reason than it being already set by `sbtTestCommand`. I guess I can check whether the stdlib compiles w/o this option.